src/atomate2/ase/jobs.py

```diff
@@ -94,10 +92,9 @@ def run_ase(
         raise NotImplementedError
 
     @property
-    @abstractmethod
     def calculator(self) -> Calculator:
         """ASE calculator, method to be implemented in subclasses."""
-        raise NotImplementedError
+        return NotImplementedError
```

Comment:
was this change accidental? can't return `NotImplementedError` here, doesn't match the return type `Calculator`. should keep the `raise` and the `@abstractmethod` decorator

Reply:
I swapped out `return` with `raise`.

I'm a bit confused on the `@abstractmethod` decorator. It seems that every forcefield that inherits the ASEMaker adapts their calculator as `@property`. Does this imply that all the calculators should be swapped from `@property` -> `@abstractmethod` ? or just the original base class where the calculator should be absent?

Reply:
whoops typo, think @BryantLi-BLI  fixed that

Reply:
> Does this imply that all the calculators should be swapped from @property -> @abstractmethod ? or just the original base class where the calculator should be absent?

just the base class should have the `@abstractmethod` decorator. but all the `calculator` methods should have the `@property` decorator

Reply:
@BryantLi-BLI : that's my bad, the `AseMaker.calculator` should be both a property and abstractmethod. I accidentally removed this in [00d1434](https://github.com/materialsproject/atomate2/pull/938/commits/00d14347e6dc653b3aab0c4457b7bf7b14a21556) (was trying to see if there was a nice way to serialize ASE calculators without having to subclass AseMaker, and just forgot to restore a few things)